std/range/package.d

```diff
@@ -948,134 +985,225 @@ if (Ranges.length > 0 &&
             }
             else
             {
-                @property bool empty()
+                @property bool empty() {return frontIndex >= backIndex;}
+            }
+
+            static if (allSatisfy!(isForwardRange, R))
+                @property auto save()
                 {
-                    foreach (i, Unused; R)
+                    auto saveI(size_t i)(){return source[i].save;}
+
+                    auto saveResult
+                        = Result(staticMap!(saveI, aliasSeqOf!(R.length.iota)));
```

Comment:
Why split this on 2 lines? I suggest keeping it on a single line. But if you insist, the `=` should be on the same line with the variable declaration.

Reply:
Trying to stay under 80 columns. But moving the equality symbol.

Reply:
I think that the <80 rule is obsolete. Screen are bigger and bigger, I would say <128 (so that it's a round number) should be the new norm.

Reply:
Maybe you're right, but we obey the old rule as long until the D style document is updated. So far it says to aim for under 80 lines, never go over 120.